config/load.go

```diff
@@ -409,7 +415,7 @@ func parseListen(cfg map[string]string, cs map[string]CertSource, readTimeout, w
 	if l.Addr == "" {
 		return Listen{}, fmt.Errorf("need listening host:port")
 	}
-	if csName != "" && l.Proto != "https" && l.Proto != "tcp" && l.Proto != "grpcs" {
+	if csName != "" && l.Proto != "https" && l.Proto != "tcp" && l.Proto != "tcp-dynamic" && l.Proto != "grpcs" {
 		return Listen{}, fmt.Errorf("cert source requires proto 'https', 'tcp' or 'grpcs'")
```

Comment:
update the error message to match the conditional

Reply:
Good catch, i'll get that updated.
Thanks!

Reply:
@leprechau, I have updated the error message to match the conditional.

Thanks!

Michael